build-scripts/multidirectory.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+import os
+import shutil
+import subprocess
+import sys
+
+def remove_branches(remote):
+    branch_refs = subprocess.check_output([
+        'git', 'branch', '--list', '--remotes', '--no-merged', 'HEAD',
+        remote + '/*'])
+    branch_names = [ref[len(remote) + 1:] for ref in branch_refs.split()]
+    for branch in branch_names:
+        prune_branch(branch)
+
+def create_or_update_branches(remote):
+    if not os.path.exists('jobs/'):
+        return
+    job_directories = \
```

Comment:
120 characters per line is fine, this stuff is only hurting the readability.

or:

```py
for directory, _, files in os.walk('jobs/'):
    if 'Jenkinsfile' in files:
        branch = job[len('jobs/'):]
        create_or_update_branch(branch)
```